#' Reimann et al., Molecular & Cellular Proteomics 2017
#'
#' Project description: The Z-disc is a protein-rich structure critically important for development and integrity of myofibrils in muscle cells.
#' Changes in protein abundance upon myoblast differentiation into myotubes and upon formation of contracting myotubes generated by electrical pulse stimulation were monitored in a quantitative proteomics study.
#' Data: Excerpts from the MaxQuant output files "proteinGroups.txt" (spreadsheet proteins); details for protein/protein group identifications are given.
#' Identifications referring to reverse hits and potential contaminants have been removed. Raw MS data and complete MaxQuant result files are available via ProteomeXchange with identifier PXD004960.
#'
#' \itemize{
#' \item Uniprot Uniprot accession numbers of all proteins assigned to the respective protein group
#' \item First_ID First ID of Protein group(s) from MaxQuant output file proteinGroups.txt
#' \item Gene_names Gene names of all proteins assigned to the respective group
#' \item Protein_names Protein names of all proteins assigned to the respective group
#' \item Main_cluster Assignment of proteins to one of the 4 main clusters after k-means clustering
#' \item Cluster Assignment of proteins to the distinct cluster after k-means clustering
#' \item Mol_weight_kDa Molecular weight in kDa
#' \item Number_of_proteins Number of proteins assigned to the respective protein group
#' \item Peptides Number of identified peptides assigned to the respective protein group
#' \item Unique_peptides Number of peptide sequences identified across all replicates of this dataset that are unique for the respective protein group
#' \item Sequence_coverage Percentage of the sequence that is covered by the identified peptides of the best protein sequence contained in the group
#' \item Mean_log10_ratio_MTP_vs_MT The mean of the log10 transformed protein ratios of paced myotubes MTP to unpaced myotubes MT
#' \item Mean_log10_ratio_MT_vs_MB The mean of the log10 transformed protein ratios of unpaced myotubes MT to myoblasts MB
#' \item Mean_log10_ratio_MTP_vs_MB The mean of the log10 transformed protein ratios of paced myotubes MTP to myoblasts MB
#' \item Minus_log10_p_value_MTP_vs_MB The negative log10 transformed p-value of paced myotubes MTP to unpaced myotubes MT
#' \item Minus_log10_p_value_MT_vs_MB The negative log10 transformed p-value of unpaced myotubes MT to myoblasts MB
#' \item Minus_log10_p_value_MTP_vs_MT The negative log10 transformed p-value of paced myotubes MTP to myoblasts MB
#' }
#'
#' @docType data
#' @keywords datasets
#' @name mcp_reimann_et_al_2017
#' @references Reimann, L., et al. Myofibrillar Z-discs Are a Protein Phosphorylation Hot Spot with Protein Kinase C (PKCalpha) Modulating Protein Dynamics. Molecular & Cellular Proteomics 2017;16(3):346-367.
#' @usage data(mcp_reimann_et_al_2017)
#' @format A data frame with 2588 rows and 17 variables
"mcp_reimann_et_al_2017"

#' Morgenstern et al., Cell Reports 2017
#'
#' Project description: Mitochondria perform central functions in cellular bioenergetics, metabolism and signaling and their malfunction has been linked to numerous diseases.
#' The available studies cover only part of the mitochondrial proteome and a separation of core mitochondrial proteins from associated fractions has not been achieved.
#' We developed an integrative, quantitative MS-based experimental approach to define the high confidence proteome of yeast mitochondria and to identify new mitochondrial proteins.
#' The analysis includes protein abundance under fermentable and non-fermentable growth, submitochondrial localization, single-protein analysis and subcellular classification of mitochondria-associated fractions.
#' We identified novel mitochondrial interactors of respiratory chain supercomplexes, ATP synthase, AAA proteases, the mitochondrial contact site and cristae organizing system (MICOS) and coenzyme Q biosynthesis cluster as well as new mitochondrial proteins with dual cellular localization.
#' The integrative proteome provides a high confidence source for the characterization of physiological and pathophysiological functions of mitochondria and their integration into the cellular environment.
#' Data: This data are a summary of proteome-wide absolute quantification of proteins under fermentable and non-fermentable growth conditions based on the 'Proteomic Ruler' approach.
#' Raw MS data and complete MaxQuant results files are available via ProteomeXchange with identifier PXD006146.
#' The variables are as follows:
#'
#' \itemize{
#' \item Systematic_names systematic name, Saccharomyces Genome Datavase (SGD)
#' \item Uniprot_ID Universal Protein database (UniProt) identifier
#' \item Gene_names name of gene
#' \item Protein_Description protein description
#' \item Moleular_weight_Da molecular weight of the leading protein sequence
#' \item High_confidence_mito_proteome reported mitochondria proteome
#' \item Sequence_coverage coverage of protein sequence by the identified peptides
#' \item Mean_copy_number_Glucose protein mean copy number in cells grown on glucose
#' \item Mean_copy_number_Galactose protein mean copy number in cells grown on galactose
#' \item Mean_copy_number_Glycerol protein mean copy number in cells grown on glycerol
#' \item Absolute_quantification_accuracy degree of quantification accuracy, according to Perseus
#' \item Copy_number_Kulak_et_al_2014 protein copy number reported by Kulak et al 2014
#' \item Copy_number_Chong_et_al_2015 protein copy number reported by Chong et al 2015
#' \item Copy_number_Ghaemmaghami_et_al_2003 protein copy number reported by Ghaemmaghami et al 2003
#' \item Cluster corresponding cluster number
#' \item pValue_ANOVA_test resulting p-value of an anova test
#' \item GOBP_terms gene ontology biological process terms
#' \item GOBP_names gene ontology biological process names
#' \item GOMF_terms gene ontology molecular function terms
#' \item GOMF_names gene ontology molecular function names
#' \item GOCC_terms gene ontology cellular component terms
#' \item GOC_names gene ontology cellular component names
#' \item Copy_number_Gal_Rep1 protein copy number based on the proteomic ruler method galactose (Gal) replicate 1
#' \item Copy_number_Gal_Rep2 protein copy number based on the proteomic ruler method galactose (Gal) replicate 2
#' \item Copy_number_Gal_Rep3 protein copy number based on the proteomic ruler method galactose (Gal) replicate 3
#' \item Copy_number_Glc_Rep1 protein copy number based on the proteomic ruler method glucose (Glc) replicate 1
#' \item Copy_number_Glc_Rep2 protein copy number based on the proteomic ruler method glucose (Glc) replicate 2
#' \item Copy_number_Glc_Rep3 protein copy number based on the proteomic ruler method glucose (Glc) replicate 3
#' \item Copy_number_Gly_Rep1 protein copy number based on the proteomic ruler method glycerol (Gly) replicate 1
#' \item Copy_number_Gly_Rep2 protein copy number based on the proteomic ruler method glycerol (Gly) replicate 2
#' \item Copy_number_Gly_Rep3 protein copy number based on the proteomic ruler method glycerol (Gly) replicate 3
#' \item Log2_copy_number_Gal_Rep1 protein copy number based on the proteomic ruler method galactose (Gal) replicate 1, log2 transformed
#' \item Log2_copy_number_Gal_Rep2 protein copy number based on the proteomic ruler method galactose (Gal) replicate 2, log2 transformed
#' \item Log2_copy_number_Gal_Rep3 protein copy number based on the proteomic ruler method galactose (Gal) replicate 3, log2 transformed
#' \item Log2_copy_number_Glc_Rep1 protein copy number based on the proteomic ruler method glucose (Glc) replicate 1, log2 transformed
#' \item Log2_copy_number_Glc_Rep2 protein copy number based on the proteomic ruler method glucose (Glc) replicate 2, log2 transformed
#' \item Log2_copy_number_Glc_Rep3 protein copy number based on the proteomic ruler method glucose (Glc) replicate 3, log2 transformed
#' \item Log2_copy_number_Gly_Rep1 protein copy number based on the proteomic ruler method glycerol (Gly) replicate 1, log2 transformed
#' \item Log2_copy_number_Gly_Rep2 protein copy number based on the proteomic ruler method glycerol (Gly) replicate 2, log2 transformed
#' \item Log2_copy_number_Gly_Rep3 protein copy number based on the proteomic ruler method glycerol (Gly) replicate 3, log2 transformed
#' \item MS_Intensity_Gal_Rep1 summed up eXtracted Ion Current (XIC) of all isotopic clusters associated with the identified AA sequence, galactose (Gal) replicate 1
#' \item MS_Intensity_Gal_Rep2 summed up eXtracted Ion Current (XIC) of all isotopic clusters associated with the identified AA sequence, galactose (Gal) replicate 2
#' \item MS_Intensity_Gal_Rep3 summed up eXtracted Ion Current (XIC) of all isotopic clusters associated with the identified AA sequence, galactose (Gal) replicate 3
#' \item MS_Intensity_Glc_Rep1 summed up eXtracted Ion Current (XIC) of all isotopic clusters associated with the identified AA sequence, glucose (Glc) replicate 1
#' \item MS_Intensity_Glc_Rep2 summed up eXtracted Ion Current (XIC) of all isotopic clusters associated with the identified AA sequence, glucose (Glc) replicate 2
#' \item MS_Intensity_Glc_Rep3 summed up eXtracted Ion Current (XIC) of all isotopic clusters associated with the identified AA sequence, glucose (Glc) replicate 3
#' \item MS_Intensity_Gly_Rep1 summed up eXtracted Ion Current (XIC) of all isotopic clusters associated with the identified AA sequence, glycerol (Gly) replicate 1
#' \item MS_Intensity_Gly_Rep2 summed up eXtracted Ion Current (XIC) of all isotopic clusters associated with the identified AA sequence, glycerol (Gly) replicate 2
#' \item MS_Intensity_Gly_Rep3 summed up eXtracted Ion Current (XIC) of all isotopic clusters associated with the identified AA sequence, glycerol (Gly) replicate 3
#' \item mean_log2_copy_number_Gal protein mean copy number grown on galactose, log2 transformed
#' \item mean_log2_copy_number_Gly protein mean copy number grown on glucose, log2 transformed
#' \item mean_log2_copy_number_Glc protein mean copy number grown on glycerol, log2 transformed
#' \item Gal_div_Glc ratio of mean log2 ratio galactose vs mean log2 ratio glucose
#' \item Gly_div_Glc ratio of mean log2 ratio glycerol vs mean log2 ratio glucose
#' }
#'
#' @docType data
#' @keywords datasets
#' @name cellr_morgenstern_et_al_2017
#' @references Morgenstern, M., et al. Definition of a High-Confidence Mitochondrial Proteome at Quantitative Scale. Cell reports 2017;19(13):2836-2852.
#' @usage data(cellr_morgenstern_et_al_2017)
#' @format A data frame with 1576 rows and 54 variables
"cellr_morgenstern_et_al_2017"

